docs/pages/dms/client.mdx

```diff
@@ -750,7 +840,21 @@ let client = try await Client.create(account: account, options: clientOptions)
       <th>Parameter</th>
       <th>Default</th>
       <th>Description</th>
-      <th></th>
+    </tr>
```

Comment:
Should the following options be added for RN?

        dbDirectory = "mydbdir", // Optional: specify database directory
        dbEncryptionKey = ByteArray(32) // Optional: 32-byte array for database

Reply:
dbEncryptionKey is not optional if you use V3

Reply:
But should be added to RN for sure.